loglist2/loglist2.go

```diff
@@ -136,6 +136,76 @@ type TreeHead struct {
 	TreeSize int64 `json:"tree_size"`
 }
 
+// String method returns printable name of the state.
+func (ls *LogStates) String() string {
+	names := [...]string{
+		"Pending",
+		"Qualified",
+		"Usable",
+		"Frozen",
+		"Retired",
+		"Rejected",
+		"Empty"}
+	if ls == nil {
```

Comment:
This section looks like it should be a switch statement
```go
switch {
	case ls.Pending:
		return names[] 
...
	default:
               return names[0] // Always use the 0'th position for empty / unknown
	}
```

Reply:
Changed.